Tangem/App/Services/ExpressPendingTransactionRepository/ExpressPendingTransactionRecord+.swift

```diff
@@ -0,0 +1,106 @@
+        date = try container.decode(Date.self, forKey: ExpressPendingTransactionRecord.CodingKeys.date)
+        externalTxId = try container.decodeIfPresent(String.self, forKey: ExpressPendingTransactionRecord.CodingKeys.externalTxId)
+        externalTxURL = try container.decodeIfPresent(String.self, forKey: ExpressPendingTransactionRecord.CodingKeys.externalTxURL)
+
+        if #unavailable(iOS 15) {
+            let feeStr = try container.decode(String.self, forKey: ExpressPendingTransactionRecord.CodingKeys.fee)
+            self.fee = Decimal(string: feeStr) ?? 0
+        } else {
+            fee = try container.decode(Decimal.self, forKey: ExpressPendingTransactionRecord.CodingKeys.fee)
+        }
+    }
+
+    func encode(to encoder: Encoder) throws {
+        var container: KeyedEncodingContainer<ExpressPendingTransactionRecord.CodingKeys> = encoder.container(keyedBy: ExpressPendingTransactionRecord.CodingKeys.self)
+
+        try container.encode(userWalletId, forKey: ExpressPendingTransactionRecord.CodingKeys.userWalletId)
+        try container.encode(expressTransactionId, forKey: ExpressPendingTransactionRecord.CodingKeys.expressTransactionId)
+        try container.encode(transactionType, forKey: ExpressPendingTransactionRecord.CodingKeys.transactionType)
+        try container.encode(transactionHash, forKey: ExpressPendingTransactionRecord.CodingKeys.transactionHash)
+        try container.encode(sourceTokenTxInfo, forKey: ExpressPendingTransactionRecord.CodingKeys.sourceTokenTxInfo)
+        try container.encode(destinationTokenTxInfo, forKey: ExpressPendingTransactionRecord.CodingKeys.destinationTokenTxInfo)
+        try container.encode(provider, forKey: ExpressPendingTransactionRecord.CodingKeys.provider)
+        try container.encode(date, forKey: ExpressPendingTransactionRecord.CodingKeys.date)
+        try container.encodeIfPresent(externalTxId, forKey: ExpressPendingTransactionRecord.CodingKeys.externalTxId)
+        try container.encodeIfPresent(externalTxURL, forKey: ExpressPendingTransactionRecord.CodingKeys.externalTxURL)
+
+        if #unavailable(iOS 15) {
+            try container.encode("\(self.fee)", forKey: ExpressPendingTransactionRecord.CodingKeys.fee)
+        } else {
+            try container.encode(fee, forKey: ExpressPendingTransactionRecord.CodingKeys.fee)
+        }
+    }
+}
+
+extension ExpressPendingTransactionRecord.TokenTxInfo {
+    enum CodingKeys: CodingKey {
+        case tokenItem
+        case blockchainNetwork
+        case amount
+        case isCustom
+    }
+
+    init(from decoder: Decoder) throws {
+        let container: KeyedDecodingContainer<ExpressPendingTransactionRecord.TokenTxInfo.CodingKeys> = try decoder.container(keyedBy: ExpressPendingTransactionRecord.TokenTxInfo.CodingKeys.self)
+
+        tokenItem = try container.decode(TokenItem.self, forKey: ExpressPendingTransactionRecord.TokenTxInfo.CodingKeys.tokenItem)
```

Comment:
```suggestion
        tokenItem = try container.decode(TokenItem.self, forKey: .tokenItem)
```
можно вот таки штуки выкинуть )

Reply:
да это Xcode нагенерил, минимум правок сделал в этот код